website/docs/commands/cli-config.html.markdown

```diff
@@ -154,8 +154,8 @@ The default way to install provider plugins is from a provider registry. The
 origin registry for a provider is encoded in the provider's source address,
 like `registry.terraform.io/hashicorp/aws`. For convenience in the common case,
 Terraform allows omitting the hostname portion for providers on
-`registry.terraform.io`, so we'd normally write `hashicorp/aws` instead in
-this case.
+`registry.terraform.io`, so public provider addresses can be written like
```

Comment:
Possible alternative `written as` ?

Reply:
Yeah, why not

```suggestion
`registry.terraform.io`, so public provider addresses can be written as
```

Reply:
Oh wait, I remember why not. It's because that's an example, not a template with placeholders, and the sentence is describing the general case. Could reword more of it, to something like: 

> ...so a public provider address of `registry.terraform.io/hashicorp/aws` can be written as `hashicorp/aws`

...but I don't think "like" is a problem. Maybe that's my west coast bias speaking, tho. 😅

Reply:
```suggestion
`registry.terraform.io`, so you can write shorter public provider addresses like
```